src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -582,4 +584,13 @@ public function getSecurityContext()
             return 'sulu.webspaces.' . $webspace->getKey();
         }
     }
+
+    /**
+     * Returns the class name of the object to check
+     * @return string
+     */
+    public function getSecuredClass()
+    {
+        return Structure::class;
```

Comment:
Should this be coupled to the controller? Why can we not infer this from the object?